examples/hysteresis.ipynb

```diff
@@ -0,0 +1,408 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9296a9ce-b102-42e6-8499-5293e03ebc69",
+   "metadata": {},
+   "source": [
+    "# Hysteresis loop simulation"
```

Comment:
Could you please add a brief summary/outline of what we will do in the notebook. Probably something like (wording needs to be improved)
- Hysteresis simulation for a single grain
- Create a simulation object
  - Set up material parameters
  - Set up simulation parameters
- Run a hysteresis loop and analyze the output